recipes/qurro/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "qurro" %}
```

Comment:
I think the style is to only use jinja templating for the `name` and `version`, and then to just put the hash directly in the `source` section below. See:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

Reply:
Done, thanks!